internal/broker/testdata/TestIsAuthenticated/golden/error_when_mode_is_password_and_token_is_offline-valid_but_server_returns_error/first_call

```diff
@@ -1,3 +1,5 @@
 access: retry
-data: '{"message": "could not authenticate user: could not load token"}'
+data: |-
+    {"message": "could not authenticate user: could not load cached info: could not refresh token: oauth2: cannot fetch token: 400 Bad Request
+    Response: "}
```

Comment:
is it expected for this to be in 2 lines or this is just the yaml formatting?

Reply:
It's just `yaml` formatting (the message gets quite large, so they switch it to a multiline string)

Reply:
thanks for confirming!